src/components/Header/index.tsx

```diff
@@ -279,14 +271,6 @@ function Header() {
 
   return (
     <HeaderFrame>
-      <ClaimModal
-        onDismiss={toggleClaimPopup}
-        newSwprBalance={newSwprBalance}
-        stakedAmount={stakedTokenAmount?.toFixed(3)}
-        singleSidedCampaignLink={
-          data && !loading ? `/rewards/single-sided-campaign/${data.stakeToken.address}/${data.address}` : undefined
-        }
-      />
```

Comment:
Why are we removing this?

Reply:
This was a request from @0xVenky as far as I understood in the ticket:
https://linear.app/swaprhq/issue/SWA-128/update-social-links-and-showcase-new-amm-concentrated-liq

![image](https://github.com/SwaprHQ/swapr-dapp/assets/21271189/f716b507-c4e4-4ce5-a9d1-a6940172fb6b)